source/common/upstream/load_balancer_impl.cc

```diff
@@ -119,6 +119,7 @@ LoadBalancerBase::LoadBalancerBase(const PrioritySet& priority_set, ClusterStats
 //   if # of healthy hosts in priority set is low.
 // - normalized total health is 0%. All hosts are down. Redirect 100% of traffic to P=0 and enable
 // panic mode.
+//   However, disable panic mode only when healthy panic threshold is 0%
```

Comment:
maybe change this comment so that on the previous bullet you say "- normalized total health is 0% and panic threshold is > 0" and phrase this line similarly?

Reply:
Yes, that's better.
Then, how about the following changes.

* Modify line 120-121:
```cpp
// - normalized total health is 0%. All hosts are down. Redirect 100% of traffic to P=0.
//   And if panic threshold > 0% then enable panic mode for P=0, otherwise disable.
```

* Delete line 122:
```cpp
// However, disable panic mode only when healthy panic threshold is 0%
```

Reply:
Sounds good to me!